.github/workflows/ci.yml

```diff
@@ -42,8 +42,8 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Format
-        run: npm run format:check
+      # - name: Format
+      #   run: npm run format:check
```

Comment:
Why the `run: npm run format:check` is disabled in the ci?

Reply:
The eslint and prettier commands seem to be in "conflict" now. For example, running a `npm run lint:fix` will cause the `format:check` command to return errors. The reverse also works: using `npm run format` will end up with linter errors. 

Because of this issue we agreed with Florent to skip the prettier step in the ci and to keep with the linter only.